build/patches/Bromite-AdBlockUpdaterService.patch

```diff
@@ -607,15 +607,15 @@ diff --git a/chrome/browser/net/system_network_context_manager.cc b/chrome/brows
 --- a/chrome/browser/net/system_network_context_manager.cc
 +++ b/chrome/browser/net/system_network_context_manager.cc
 @@ -343,6 +343,8 @@ SystemNetworkContextManager::SystemNetworkContextManager(
-           SSLConfigServiceManager::CreateDefaultManager(local_state_)),
-       proxy_config_monitor_(local_state_),
        stub_resolver_config_reader_(local_state_) {
+ #if !BUILDFLAG(IS_ANDROID)
+   // QuicAllowed was not part of Android policy.
 +  local_state_->SetDefaultPrefValue(prefs::kAdBlockFiltersURL,
 +                                    base::Value("https://www.bromite.org/filters/filters.dat"));
```

Comment:
the position is wrong, look above `#if !BUILDFLAG(IS_ANDROID)`

Reply:
This was done by patch fuzzying.